lighthouse/Cargo.toml

```diff
@@ -81,6 +81,7 @@ malloc_utils = { workspace = true, features = ["jemalloc"] }
 malloc_utils = { workspace = true, features = [] }
 
 [dev-dependencies]
+beacon_node = { workspace = true, features = ["testing"] }
```

Comment:
I think this works OK because we're using Cargo's v2 resolver (https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2). I guess we need this for the CLI tests involving these flags